.github/workflows/basic.yml

```diff
@@ -15,12 +15,12 @@ on:
 env:
   GENOMICSDB_BUILD_DIR: ${{github.workspace}}
   GENOMICSDB_HOME: ${{github.workspace}}/install
-  GENOMICSDB_BRANCH: develop
+  GENOMICSDB_BRANCH: ng_nanoarrow
```

Comment:
Will revert to develop once https://github.com/GenomicsDB/GenomicsDB/pull/338 is approved.